ui/packages/consul-ui/app/adapters/binding-rule.js

```diff
@@ -0,0 +1,28 @@
+import Adapter from './application';
+
+export default class BindingRuleAdapter extends Adapter {
+  requestForQuery(request, { dc, ns, authmethod, index, id }) {
+    return request`
+      GET /v1/acl/binding-rules?${{ dc, authmethod }}
```

Comment:
@johncowen the filtering doesn't work when I camel case `authmethod`. [Documentation](https://www.consul.io/api-docs/acl/binding-rules#authmethod-2)